datafusion/sqllogictest/test_files/insert_to_external.slt

```diff
@@ -276,10 +270,7 @@ statement ok
 CREATE EXTERNAL TABLE
 directory_test(a bigint, b bigint)
 STORED AS parquet
-LOCATION 'test_files/scratch/insert_to_external/external_parquet_table_q0'
-OPTIONS(
-create_local_path 'true',
-);
+LOCATION 'test_files/scratch/insert_to_external/external_parquet_table_q0/';
```

Comment:
I'm not actually sure how this was working before, the lack of a trailing `/` would describe a file location not a directory